device/inventec/x86_64-inventec_d6332-r0/led_proc_init.soc

```diff
@@ -0,0 +1,6 @@
+#led auto off
+#led stop
```

Comment:
Commented out lines. Remove?

Reply:
@jleveque Hi Joe, I have removed the commented lines in "led_proc_init.soc" and submitted a new commit. These lines were commented out since they will fail and the following "m0 load *" commands won't be executed.  Please let me know if you have any other questions. Thanks.